src/Demos/HSP.lagda

```diff
@@ -371,56 +313,53 @@ module _  {𝑨 : Setoid α ρᵃ}{𝑩 : Setoid β ρᵇ}{𝑪 : Setoid γ ρ
   Goal : Image g ⟨∘⟩ f ∋ y
   Goal = mp gonto
 \end{code}
-\fi      %%% END LONG VERSION ONLY SECTION
+\fi
 
 \paragraph*{Kernels of setoid functions}
 The \defn{kernel} of a function \ab f~\as :~\ab A~\as{→}~\ab B (where \ab A and \ab B are bare types) is defined
 informally by \{\AgdaPair{x}{y} \aod{∈} \ab A \aof{×} \ab A \as : \ab f \ab x \as{=} \ab f \ab y \}.
-This can be represented in \agda in a number of ways, but for our purposes it is most
-convenient to define the kernel as an inhabitant of a (unary) predicate over the square of
+This can be represented in \agda in a number of ways, but for our purposes it is
+convenient to define the kernel as inhabitant of a (unary) predicate over \ab A \aof{×} \ab A where \ab is
```

Comment:
Two typos here:
1. "as inhabitant" should be "as an inhabitant"
2. "where \ab is" should be "where \ab A is"